documentation/library_guide/api_for_dpcpp_kernels/tested_standard_cpp_api.rst

```diff
@@ -358,6 +358,18 @@ C++ Standard API                     libstdc++  libc++     MSVC
 ``std::move``                        Tested     Tested     Tested
 ------------------------------------ ---------- ---------- ----------
 ``std::move_backward``               Tested     Tested     Tested
+------------------------------------ ---------- ---------- ----------
+``std::abs``                         Tested     Tested     Tested
+------------------------------------ ---------- ---------- ----------
+``std::arg``                         Tested     Tested     Tested
+------------------------------------ ---------- ---------- ----------
+``std::conj``                        Tested     Tested     Tested
```

Comment:
I'd put these functions next to `std::complex` in the list, as they are specific for the complex type. Same for `imag` and `real` below.

Reply:
Fixed